packages/prisma/src/prisma/schema/element.prisma

```diff
@@ -31,7 +31,7 @@ model Element {
 
   version Int @default(1) // used to track question versions, incremented on each update
 
-  originalId String? @unique
+  originalId String?
```

Comment:
This is desired behavior, since the user would otherwise not be able to create new questions anymore. The `originalId` prop is only designed to be used for the mgiration process from KlickerUZH v2